md2conf/converter.py

```diff
@@ -317,6 +317,12 @@ def _transform_link(self, anchor: ET._Element) -> None:
         LOGGER.debug(f"found link {url} relative to {self.path}")
         relative_url: ParseResult = urlparse(url)
 
+        if relative_url.fragment and not relative_url.path:
+            # This is a local anchor link
+            LOGGER.debug(f"found local anchor URL: {url}")
+            anchor.attrib["href"] = f"ac:link ac:anchor='{relative_url.fragment}'"
+            return
```

Comment:
This seems like magic. Why would we want to inject a series of qualified attribute name and value pairs into the `href` attribute of the anchor element? If this works, it appears that we are exploiting a bug in `lxml`. I assume that we might want to construct a new element (either an anchor or a structured macro) and populate the fields based on what attributes the original anchor element had. There are a few examples in the source code how to do this.